parameter-object/src/test/java/com/iluwatar/parameter/object/AppTest.java

```diff
@@ -23,19 +23,16 @@
 
 package com.iluwatar.parameter.object;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
 import org.junit.jupiter.api.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * Application test
  */
 class AppTest {
-  private static final Logger LOGGER = LoggerFactory.getLogger(AppTest.class);
-
   @Test
   void shouldExecuteApplicationWithoutException() {
-    App.main(new String[]{});
-    LOGGER.info("Executed successfully without exception.");
```

Comment:
Any reasons for removing logging? Was it showing any issues on sonar?

Reply:
The purpose of the logging was to assert that the code executed without errors. Since there is an aproprieted assertion for that, there is no need for this logging.